oblt-cli/cluster-create-ccs/action.yml

```diff
@@ -0,0 +1,114 @@
+name: 'Oblt-cli create ccs'
+description: 'Run the oblt-cli wrapper to create a CCS cluster.'
+inputs:
+  remote-cluster:
+    description: 'The Oblt cluster to use'
+    required: true
+  github-token:
+    description: 'The GitHub access token.'
+    required: true
+  cluster-name-prefix:
+    description: 'Prefix to be prepended to the randomised cluster name'
+    required: false
+  cluster-name-suffix:
+    description: 'Suffix to be appended to the randomised cluster name'
+    required: false
+  elasticsearch-docker-image:
+    description: 'Force to use a Docker image for the Elasticserach Deployment'
+    required: false
+  slack-channel:
+    description: 'The slack channel to notify the status.'
+    default: '#observablt-bots'
+    required: false
+  username:
+    description: 'Username to show in the deployments with oblt-cli, format: [a-z0-9]'
+    default: 'apmmachine'
+    required: false
+  gitops:
+    description: 'Whether to provide the GitOps metadata to the oblt-cli'
+    default: false
+    required: false
+  dry-run:
+    description: 'Whether to dryRun'
+    default: false
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Configure oblt-cli flags
```

Comment:
Wonder whether this snippet should be a shared one between all the `oblt-cli/cluster-create*`

Reply:
I guess we can create a separate action for flags
Let me do that in a follow-up since there are some minor differences in each script that we need to make generic